centreon-connector/perl/src/embedded_perl.cc

```diff
@@ -231,8 +226,8 @@ pid_t embedded_perl::run(std::string const& cmd, int fds[3]) {
     call_pv("Embed::Persistent::run_file", G_DISCARD);
     std::cerr << "error while executing Perl script '" << file
               << "': " << SvPV_nolen(ERRSV) << std::endl;
-    exit(3);
-    abort();
+    //    exit(3);
+    //   abort();
```

Comment:
Is it wanted?
* Did you forget to remove comments?
* Did you forget to remove these lines?

Reply:
done